### PR TITLE
Update to specs-actors v0.9.8

### PR DESCRIPTION
**Pull request by arajasek**
This is actually fully backwards-compatible with v0.9.7, so we could just merge into master, but might as well send it into the staging branch.

### CODE REVIEW

File: chain/gen/genesis/genesis.go [hunk 1]
```diff
@@ -406,8 +406,8 @@ func VerifyPreSealedData(ctx context.Context, cs *store.ChainStore, stateroot ci
 	verifNeeds := make(map[address.Address]abi.PaddedPieceSize)
 	var sum abi.PaddedPieceSize
 
-	nwv := func(context.Context, abi.ChainEpoch) runtime.NetworkVersion {
-		return runtime.NetworkVersion1
+	nwv := func(context.Context, abi.ChainEpoch) network.Version {
+		return network.Version1
```
Comment (magik6k) [inline chain/gen/genesis/genesis.go:410]: Not Version2?

Also, I feel like this belongs in `build/`, `NewNetworkVersion` or something like that

File: chain/stmgr/stmgr.go [hunk 2]
```diff
@@ -1124,14 +1124,14 @@ func (sm *StateManager) GetCirculatingSupply(ctx context.Context, height abi.Cha
 	return csi.FilCirculating, nil
 }
 
-func (sm *StateManager) GetNtwkVersion(ctx context.Context, height abi.ChainEpoch) runtime.NetworkVersion {
+func (sm *StateManager) GetNtwkVersion(ctx context.Context, height abi.ChainEpoch) network.Version {
 	if build.UpgradeBreezeHeight == 0 {
```
Comment (magik6k) [inline chain/stmgr/stmgr.go:1128]: It may be hardcoded at -1 now for devnets

File: chain/gen/genesis/genesis.go (hunk 1 above)
Comment (arajasek) [reply chain/gen/genesis/genesis.go:410]: No, we're still version 1 (from the PoV of actors, which is what cares about this).

Comment (magik6k) [reply chain/gen/genesis/genesis.go:410]: `NewestNetworkVersion`?

**Review by magik6k — APPROVED** [review]
1 Nit

File: chain/gen/genesis/genesis.go (hunk 1 above)
Comment (arajasek) [reply chain/gen/genesis/genesis.go]: Yeeee, fixed (and also in genesis miners setup)

**Comment by arajasek** [comment]
Merging so we can cut an RC, but post-merge review still welcome

**Pull request merged**